array.js

```diff
@@ -140,3 +140,83 @@ const getMaxElementArray = (array) => {
 };
 
 console.log(getMaxElementArray(arr));
+
+/*  */
+
+/* 1. Дано масив з елементами різних типів. Створити функцію яка вираховує середнє арифметичне лише числових елементів даного масиву. */
+
+const array = [{ value: 96 }, 4, 'split', 7, { vw: 'Jetta', number: 1841, region: 'СВ' }, '4', 'John'];
+
+function getAverageNumber(arr) {
+	if (!Array.isArray(arr)) {
+		throw new Erro('Error: Input parameter must be an array.');
+	}
+	const numbers = arr.filter((item) => typeof item === 'number');
+	const sum = numbers.reduce((a, b) => a + b);
+	return sum / numbers.length;
+}
+
+console.log(getAverageNumber(arr));
+
+/* 2. Написати функцію doMath(x, znak, y), яка отримує 3 аргументи: числа x і y, рядок znak. У змінній znak може бути: +, -, *, /, %, ^ (ступінь ).Вивести результат математичної дії, вказаної в змінній znak.Обидва числа і знак виходять від користувача. */
+
+const doMath = (x, y, znak) => {
+	if ((typeof x === 'number', typeof y === 'number', typeof znak === 'string')) {
+		switch (znak) {
+			case '+':
+				return x + y;
+			case '-':
+				return x - y;
+			case '*':
+				return x * y;
+			case '/':
+				return x / y;
+			case '%':
+				return x % y;
+			case '^':
+				return x ** y;
+
+			default:
+				console.log('Error: Parameter not found.');
+		}
+	}
+};
```

Comment:
Instead of using comma operator in the if statement, you should use logical AND (&&) operator to combine the conditions. The way you have it written now will only check the last condition (typeof znak === 'string') and ignore the other two conditions.

Instead of logging an error message to the console, you should throw an error using the throw keyword. This way, if there's an error, the calling code can catch the error and handle it appropriately.

also naming of parameters is bad
1. `a, b` instead of `x, y` (`x, y` is used to determine the horizontal and vertical coordinates, so it is not appropriate here)
2. ʼznakʼ should be in English `sign`